osu.Game/Screens/OnlinePlay/Multiplayer/Match/MultiplayerReadyButton.cs

```diff
@@ -55,7 +55,21 @@ protected override void LoadComplete()
 
         private void onRoomUpdated() => Scheduler.AddOnce(() =>
         {
-            if (countdown != room?.Countdown)
+            MultiplayerCountdown newCountdown;
+
+            switch (room?.Countdown)
+            {
+                case MatchStartCountdown _:
+                    newCountdown = room.Countdown;
+                    break;
+
+                // Clear the countdown with any other (including non-null) countdown values.
+                default:
+                    newCountdown = null;
+                    break;
```

Comment:
~~I'm not sure about this... is it required to make things work for some reason? Feels like it will fail in the future if other countdowns are added.~~

It seems that a room can only have one active countdown at one. I'm not sure how well this will play out into the future, but I guess this is okay for now..?